lib/cocoapods/target/pod_target.rb

```diff
@@ -427,9 +427,9 @@ def module_map_path
       basename = "#{label}.modulemap"
       if build_as_framework?
         super
-      elsif file_accessors.any?(&:module_map)
```

Comment:
this is when a custom module map is present by the podspec? can you not place it into the folder you expect it to be placed?